src/contacts/contact-base.cpp

```diff
@@ -21,18 +21,18 @@ namespace tsid
 {
   namespace contacts
   {
-    ContactBase::ContactBase(const std::string & name,
-                             RobotWrapper & robot):
-      m_name(name),
-      m_robot(robot)
-    {}
+    ContactBase::ContactBase(const std::string &name,
+                             RobotWrapper &robot) : m_name(name),
+                                                    m_robot(robot)
+    {
+    }
 
-    const std::string & ContactBase::name() const
+    const std::string &ContactBase::name() const
     {
       return m_name;
     }
 
-    void ContactBase::name(const std::string & name)
+    void ContactBase::name(const std::string &name)
```

Comment:
there are still cosmetic changes everywhere and honestly I am getting tired of reviewing this PR over and over again finding always the same issue.

Reply:
@olivier-stasse I am fine with what you suggested, but I don't have the time to set this up right now. As far as I know @nim65s is on paternity leave in this moment, so I don't know who could take charge of this.

Reply:
I can do if this is fine with you.

Reply:
Sure, that'd be great, thanks!

Reply:
#197 implements the clang-format in the pre-commit